rkt/status.go

```diff
@@ -30,7 +30,7 @@ import (
 
 var (
 	cmdStatus = &cobra.Command{
-		Use:   "status [--wait=bool|timeout] [--wait-ready=bool|timeout] UUID",
+		Use:   "status [--wait=bool|timeout] [--wait-ready=bool|timeout] --uuid-file=FILE | UUID ...",
```

Comment:
Why the `...`? This seems to indicate you can input several UUIDs, but reading the code that's not the case.